assets/stylesheets/components/_alerts.scss

```diff
@@ -0,0 +1,27 @@
+.alert {
+  border-radius: 0;
+  line-height: 1.125rem;
+
+
+  &.alert-success {
+    background: lighten($brand-success, 40);
```

Comment:
Can we refactor this to use Bootstrap's `alert-variant()` mixin?

https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_alerts.scss